src/seq/bioseq.jl

```diff
@@ -63,20 +63,48 @@ typealias CharSequence      BioSequence{CharAlphabet}
 
 remove_newlines(s) = replace(s, r"\r|\n", "")
 
-macro dna_str(seq, flags...)
-    return DNASequence(remove_newlines(seq))
+macro dna_str(seq, flag)
```

Comment:
I think we should overload the `dna_str` macro with `dna_str(seq)` (no `flag` argument) as well to recover the original behavior.